lib/utilities/toUTC.py

```diff
@@ -2,14 +2,15 @@
 from dateutil.parser import parse
 import pytz
 import math
+import tzlocal
+
+local_timezone = tzlocal.get_localzone()
```

Comment:
I do question if there's a better way to do this. I didn't want that get_localzone() function called every time toUTC is called however...maybe a check for if it's None, set it?

Reply:
The only other way I know how to do it is via something like: 
https://github.com/jeffbryner/ldapChangeMonitor/blob/master/ldapChangeMonitor.py#L244

It would be interesting to profile.